enterprise/com/src/main/java/org/neo4j/com/storecopy/TransactionCommittingResponseUnpacker.java

```diff
@@ -230,6 +231,10 @@ public void obligation( long txId ) throws IOException
             {
                 obligationFulfiller.fulfill( txId );
             }
+            catch ( IllegalStateException e )
+            {
+                throw new ComException( "Failed to pull updates", e );
+            }
             catch ( InterruptedException e )
             {
                 throw new IOException( e );
```

Comment:
You only re-interrupt if you catch and ignore an InterruptedException, typically. Here we're re-throwing as an IOException, so I think it's fine as is.